src/module/item/base/document.ts

```diff
@@ -688,6 +688,9 @@ class ItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Item
 
         // Convert all non-kit sources to item objects, and recursively extract the simple grants from ABC items
         const actorClone = actor.clone({}, { keepId: true });
+        for (const [key, value] of actor._dependentTokens.entries()) {
+            actorClone._dependentTokens.set(key, value);
+        }
```

Comment:
Any reason to not do this in a `clone` override?

Reply:
Done!